bindings/experimental/distrdf/python/DistRDF/HeadNode.py

```diff
@@ -358,15 +358,18 @@ def __init__(self, backend: BaseBackend, npartitions: Optional[int], *args):
         super().__init__(backend, npartitions)
 
         self.defaultbranches = None
-        self.friendinfo = None
+        # Information about friend trees. It is not None only if the TTree
+        # constructor overload is used and if there is at least one friend.
+        self.friendinfo: Optional[ROOT.Internal.TreeUtils.RFriendInfo] = None
 
         # Retrieve the TTree/TChain that will be processed
         if isinstance(args[0], ROOT.TTree):
             # RDataFrame(tree, defaultBranches = {})
             self.tree = args[0]
             # Retrieve information about friend trees when user passes a TTree
             # or TChain object.
-            self.friendinfo = ROOT.Internal.TreeUtils.GetFriendInfo(args[0])
+            _fi = ROOT.Internal.TreeUtils.GetFriendInfo(args[0])
```

Comment:
the leading underscore confused me for a sec there :thinking: local variables are by construction internal/private, why does this one deserve a `_` and other local variables in other methods do not?

Reply:
You are right, my brain decided to promote this to a private variable for some reason :)